src/vs/workbench/contrib/terminal/common/scripts/shellIntegration.ps1

```diff
@@ -169,6 +169,14 @@ elseif ((Test-Path variable:global:GitPromptSettings) -and $Global:GitPromptSett
 	[Console]::Write("$([char]0x1b)]633;P;PromptType=posh-git`a")
 }
 
+
+if (-not (Get-Module -Name PSReadLine) -and $env:VSCODE_A11Y_MODE) {
+	$scriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path
+	$specialPsrlPath = Join-Path $scriptRoot '\psreadline\'
+	Import-Module $specialPsrlPath
+}
```

Comment:
We should be unsetting `VSCODE_A11Y_MODE` like we do here:

https://github.com/microsoft/vscode/blob/e921a2ee51491b6b2b3825a7eded370760acc624/src/vs/workbench/contrib/terminal/common/scripts/shellIntegration.ps1#L29-L33